homework-submission/barbarosapp/src/components/itemList.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Item from "./item";
+
+class ItemList extends React.Component {
+  render() {
+    const { Data, handleCheck } = this.props;
+
+    const createItems = Data.map(item => {
+      return (
+        <section className="ToDoItem">
+          <Item
+            key={item.id}
+            id={item.id}
+            name={item.name}
+            deadLine={item.deadline}
+            done={item.done}
+            handleCheck={() => handleCheck(item.id)}
+          />
+        </section>
+      );
+    });
+
+    const emptyCheck =
+      Data.length > 0 ? (
+        createItems
+      ) : (
+        <div className="ToDoList">
+          <h2>Your List is EMPTY</h2>
+        </div>
+      );
+
+    return <div className="ToDoList">{emptyCheck}</div>;
```

Comment:
it would be easier to read this as the following 🙂
from line 23 till this line:
```suggestion
    const emptyCheck = Data.length > 0 ? createItems : <h2>Your List is EMPTY</h2>;
    
    return <div className="ToDoList">{emptyCheck}</div>;
```
or other way:
```suggestion
    const EmptyCheck = () => Data.length > 0 ? createItems : <h2>Your List is EMPTY</h2>;
    
    return (
      <div className="ToDoList">
        <EmptyCheck />
      </div>
    );
```

Reply:
Corrected at item.js of week 3 homework.